stories/notifications.stories.js

```diff
@@ -224,6 +224,10 @@ storiesOf( 'Global/Notifications', module )
 		/>
 	) )
 	.add( 'User Input Settings', () => {
+		global._googlesitekitUserData = {
+			user: {},
+			userInputState: 'missing',
+		};
```

Comment:
Instead of using this rather hacky approach, it would be better to provide the information via the internal datastore actions. `provideUserInfo` already takes care of the `user` object via the `receiveUserInfo` action, but it right now doesn't call `receiveConnectURL`, `receiveUserIsVerified`, and `receiveUserInputState`, which is why the error in Storybook happens.

Can you replace this with a call to `receiveUserInputState( 'missing' )` for now? With that you should be good, since `provideUserInfo` takes care of the user object.

Reply:
@felixarntz Thank you for the detailed explanation, this makes a bit more sense to me now. Adding `receiveUserInputState` fixes the issues I mentioned above and is a much cleaner approach.